docs/reference/commandline/history.md

```diff
@@ -54,3 +55,41 @@ IMAGE               CREATED             CREATED BY
 c69cab00d6ef        5 months ago        /bin/sh -c #(nop) MAINTAINER Lokesh Mandvekar   0 B
 511136ea3c5a        19 months ago                                                       0 B                 Imported from -
 ```
+
+### Format the output
+
+The formatting option (`--format`) will pretty print history output
+using a Go template.
+
+Valid placeholders for the Go template are listed below:
+
+| Placeholder | Description|
+| ---- | ---- |
+| `.ID` | Image ID |
+| `.CreatedSince` | Elapsed time since the image was created |
```

Comment:
It seems both `Created` and `CreatedSince` has been defined for placeholders. Though here the `Created` is not mentioned in the docs.

I am wondering if we add both in the docs, or, maybe we could just merge `Created` and `CreatedSince` into one placeholder func?